api/src/shared/infrastructure/utils/logger.js

```diff
@@ -101,13 +83,19 @@ function _logInfoWithCorrelationIds(logger) {
  * logErrorWithCorrelationIds(data);
  *
  * @param {object} data
+ * @deprecated use logger#error instead
+ */
+const logErrorWithCorrelationIds = _logWithCorrelationIds(internalLogger.error);
+
+/** @param {object} data
+ * Usage : show logErrorWithCorrelationIds
+ * @deprecated use logger#info instead
  */
-const logErrorWithCorrelationIds = _logErrorWithCorrelationIds(internalLogger);
-const logInfoWithCorrelationIds = _logInfoWithCorrelationIds(internalLogger);
+const logInfoWithCorrelationIds = _logWithCorrelationIds(internalLogger.info);
 
 const logger = {
-  error: logErrorWithCorrelationIds,
-  info: logInfoWithCorrelationIds,
+  error: _logWithCorrelationIds(internalLogger.error),
+  info: _logWithCorrelationIds(internalLogger.info),
```

Comment:
suite aux remontées d'erreur de la CI:

```
/home/circleci/pix/api/node_modules/pino/lib/tools.js:59
      if (typeof this[msgPrefixSym] === 'string' && msg !== undefined && msg !== null) {
```

Je pense qu'il faudrait plutôt faire comme ça

```suggestion
  error: _logWithCorrelationIds(internalLogger.error.bind(internalLogger)),
  info: _logWithCorrelationIds(internalLogger.info.bind(internalLogger)),
```